Portland 2021 schedule - provisional

plaindocs opened the pull request:
- [x] needs blog post
- [ ] needs confirmation from speakers
- [x] some times to fix
   ![image](https://user-images.githubusercontent.com/95874/110469105-81ddfe00-80d9-11eb-940b-7f4243b0d572.png)

plaindocs committed:
Fix unrelated title underlines

plaindocs committed:
Provisional schedule

plaindocs committed:
Use TZ variable

ericholscher commented on docs/conf/portland/2021/news/cfp-opens.rst:
```diff
@@ -19,7 +19,7 @@ We're looking for ideas and use-cases from all disciplines and roles, so whether
 
 You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.
 
-The Call for Proposals will be open until **Midnight PST {{cfp.ends}}**.
```
The timezone changes from PST to PDT during our news cycle 🤯

plaindocs replied:
Valid point, although what matters is the timezone it is in when the date occurs, right?

ericholscher commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,73 @@
+  - time: '9:45'
+    title: Team up and start working
+  - time: '12:30'
+    title: Lunch Break
```
Should probably include a `Return from lunch` or something to note the timing, but not a huge deal.

ericholscher commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,73 @@
+  - time: '9:45'
+    slug: writing-a-perfect-technical-tutorial-jessica-garson
+  - time: '10:30'
+    slug: shuffle-ball-change-sashay-your-way-to-clearer-api-documentation-rachael-stavchansky
```
I think we should have a break, even for the virtual conference. Otherwise it feels a bit overwhelming.

plaindocs replied:
Done

ericholscher commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,73 @@
+    title: Lunch (60 mins)
+  - time: '1:45'
+    title: Lightning Talks (30 mins)
+  - time: '2:15'
```
We should add a buffer after the lightning talks.

ericholscher commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,73 @@
+  - time: '3:00'
+    slug: putting-the-tech-in-technical-writer-swapnil-ogale
+  - time: '3:45'
+    slug: always-complete-never-finished-daniele-procida
```
Feels late for someone in Europe, no?

plaindocs replied:
:+1: I always forget that Daniele is in Europe.

ericholscher commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,73 @@
+  - time: '1:45'
+    title: Lightning Talks (30 mins)
+  - time: '2:15'
+    title: "<b>Unconference Starts</b>"
```
This feels weak. If we only have 2 afternoon sessions, we should probably do something different w/ the unconf. Not 100% sure what tho.

I'd propose we move lunch forward to 12, which would balance things more nicely.

ericholscher reviewed (commented):
Good start, definitely needs some tweaks.

ericholscher commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,73 @@
+    slug: level-up-onboarding-that-enables-writers-to-thrive-nicola-yap
+  - time: '3:45'
+    title: Closing address
+  - time: '4:30'
```
This feels long for the closing with no Q&A. I'd say make the closing 15.

ericholscher commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,73 @@
+  - time: '1:45'
+    title: Lightning Talks (30 mins)
+  - time: '2:15'
+    slug: the-secret-history-of-libraries-paris-buttfield-addison
```
Same re: europe timing.

plaindocs replied:
And of course Paris is in Australia. :+1:

ericholscher commented:
@plaindocs do you want me to push a commit w/ a revised schedule with my comments?

plaindocs committed:
All change

thatdocslady commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,79 @@
+writing_day:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:30'
```
I think we pushed this back to 9:00 to save some time?

thatdocslady commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,79 @@
+  - time: '12:30'
+    title: Lunch Break
+  - time: '5:00'
+    title: End of Writing Day
```
We should check with @canncrochet what are the WW tour/orientation plans and put them on the schedule (Hi Christy! Any input here?)

thatdocslady commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,79 @@
+    slug: putting-the-tech-in-technical-writer-swapnil-ogale
+  - time: '4:30'
+    slug: the-secret-history-of-libraries-paris-buttfield-addison
+  - time: '5:15'
```
Do we want to have the wrap-up prerecorded and inserted before the final Q&A? How would we indicate this?

ericholscher replied:
This is a good question, not sure the best way to handle it in the schedule.

plaindocs replied:
I can't remember why we wanted to do this?

ericholscher replied:
Because otherwise we have to herd people back to the main stage for the closing which feels weird, especially if you don't want the Q&A.

I think the main argument against is that the folks in the unconf rooms might miss it if it isn't scheduled, but not 100% sure whats best.

plaindocs replied:
Yeah. If they haven't got it by the end of the day...

But otherwise, we could add "& QA" to *each* talk, and add a specific "Q&A for X talk" line after?

plaindocs replied:
To be honest, after thinking this through, I think it is a non issue. Folks are gonna watch what they want either way. 

* Structure like any other talk + Q&A, final Outro back on the main stage (link will be there so folks can watch if they miss it?)
* Add a single 5 min Q&A slot after the wrap up (on the cal) and add a mention of it to the MC intro for the final talk

Either seems fine to me.

ericholscher replied:
I think we don't need to decide now. People aren't watching the schedule so closely that they'll notice any changes like this around the closing. Let's ship it 👍

thatdocslady commented on docs/_data/portland-2021-schedule.yaml:
```diff
@@ -0,0 +1,79 @@
+    slug: invisible-influence-the-documentation-behind-ux-copy-katherine-karaus
+  - time: '3:30'
+    slug: level-up-onboarding-that-enables-writers-to-thrive-nicola-yap
+  - time: '4:45'
```
Same comment re/ closing announcements and final Q&A

ericholscher committed:
Adjustments to the schedule

ericholscher committed:
Add small script for generating schedule

mxsasha commented:
Just one thing on a0af185, I don't understand the purpose of that script? It's also seems very specific to a particular workflow that is not clear from the script. This is exactly the kind of thing I've been trying to avoid with the scripts cleanup last year ;)

ericholscher commented:
@mxsasha it converts from the google doc CSV output into the YAML. Otherwise updating/changing times on the schedule is a huge pain.

plaindocs commented:
I'll add some info to the script before we merge. Cheers both.

plaindocs committed:
Document script

plaindocs committed:
Incomplete MC info script

thatdocslady committed:
Portland 2021 schedule blog post (#1461)

* Add additional sponsors (#1456)

* Add additional sponsors

* Clean up spnosor text

* Removing mention of meals from ticket price (#1458)

* Add conferences link and remove redundant links (#1460)

* Correct minor typos and update Jinja link

* Add conferences link and remove redundant links

docs/guide/index.rst

- Condense intro contribution wording
- Remove redundant link to contributing guidelines

docs/guide/contributing.rst

- Update Slack link
- Replace old forums link with conferences link

* Portland 2021 schedule blog post

Co-authored-by: Eric Holscher <25510+ericholscher@users.noreply.github.com>
Co-authored-by: Janine Chan <64388808+janine-c@users.noreply.github.com>
Co-authored-by: traveldwindling <74505760+traveldwindling@users.noreply.github.com>

ericholscher commented on docs/conf/portland/2021/news/announcing-schedule.rst:
```diff
@@ -0,0 +1,61 @@
+
+One of the big changes in the schedule based on the community feedback is scheduling more breaks during the day. We are all trying to balance our screen time, so we wanted to make sure that your eyes get their rest! We hope that this adjustment will help you stay alert and happy throughout the conferences.
+
+Another change that we made is to move the social event to an earlier time and shorten the activity time slots. We want you to have fun socializing with your fellow documentarians, but we understand that your family and home time is also important these days, so hopefully this will help us stay engaged and enjoy the social activities even more.
```
What are activity time slots?

plaindocs replied:
```suggestion
Another change that we made is to move the social event to an earlier time. We want you to have fun socializing with your fellow documentarians, but we understand that your family and home time is also important these days, so hopefully this will help us stay engaged and enjoy the social activities even more.
```

ericholscher commented on docs/conf/portland/2021/news/announcing-schedule.rst:
```diff
@@ -0,0 +1,61 @@
+Conference platform
+--------------------
+
+Following the success of last year's virtual format, the conference will again be held on https://hopin.to/. Hopin not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and expo booths. We're updating our Welcome Wagon guide, to be released soon, so it'll all feel familiar before you've even arrived.
```
Should we announce the social event platform here? https://spatial.chat/

plaindocs replied:
I feel like you're the only person who knows much about it. Wanna add a para? @ericholscher

ericholscher commented on docs/conf/portland/2021/news/announcing-schedule.rst:
```diff
@@ -0,0 +1,61 @@
+* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
+* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place.
+* **Job Fair**. As with the previous few years, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.
+* **Social event**. On Monday night the conference platform will become the social event space. The main stage will be closed, the main focus will be attendee-created sessions. You're welcome to run sessions for board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.
```
```suggestion
* **Social event**. On Monday night we'll be moving to a spatial chat platform for our social event. This will be a more familiar social space where you can move around and join groups of people having conversations. We've used similar platforms at other events, and found that they really help people get to know each other in a more natural way. 
```

plaindocs committed:
Apply suggestions from code review

Co-authored-by: Eric Holscher <25510+ericholscher@users.noreply.github.com>

plaindocs commented:
OK this is good. We should wait a day or so for speakers to confirm though.

thatdocslady commented on docs/conf/portland/2021/news/announcing-schedule.rst:
```diff
@@ -0,0 +1,61 @@
+One of the big changes in the schedule based on the community feedback is scheduling more breaks during the day. We are all trying to balance our screen time, so we wanted to make sure that your eyes get their rest! We hope that this adjustment will help you stay alert and happy throughout the conferences.
+
+Another change that we made is to move the social event to an earlier time. We want you to have fun socializing with your fellow documentarians, but we understand that your family and home time is also important these days, so hopefully this will help us stay engaged and enjoy the social activities even more.
+
```
```suggestion

**All talk videos will be published on our YouTube channel no later than 1 week after the conference.** 
```

thatdocslady replied:
HERE IT IS IN BOLD

ericholscher replied:
I think we need it on the website as well.

ericholscher commented on docs/conf/portland/2021/news/announcing-schedule.rst:
```diff
@@ -0,0 +1,61 @@
+* **Conference Help Desk**. If you have any questions about navigating the virtual event, we'll be running the Write the Docs Helpdesk booth, where you can chat with one of the organizers and the Welcome Wagon team.
+
+The `Schedule page <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ has the latest and most complete information about all our events. This is where we will also update any changes/cancellations if they should occur.
+
```
We should thank our sponsors at well in this post.

ericholscher committed:
Add sponsors to announcement & change date to today

ericholscher committed:
Update docs/conf/portland/2021/news/announcing-schedule.rst

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

ericholscher committed:
Mention talk videos on the website a few places :)

ericholscher reviewed (approved):
🎉

ericholscher committed:
Merge branch 'master' into portland-2021-schedule
